ufm-test-services/.env.example

```
@@ -1,10 +1,19 @@
 CI=false
 
-GRAFANA_ADMIN_PWD=
+GRAFANA_ADMIN_PWD=op
+
+PROMETHEUS_SERVER_URL="http://prometheus:9090"
+PROMETHEUS_PUSHGATEWAY_URL="http://pushgateway:9091"
 
 # Used to set the display to be used by playwright when running Metamask test service.
 # The following are to be used if running on MacOS
-# MM_DISPLAY=host.docker.internal:0
-# MM_DISPLAY_VOLUME=/tmp/.X11-unix:/tmp/.X11-unix
-MM_DISPLAY=
-MM_DISPLAY_VOLUME=
+METAMASK_RUN_HEADLESS=true
+# METAMASK_DISPLAY=host.docker.internal:0
+# METAMASK_DISPLAY_VOLUME=/tmp/.X11-unix:/tmp/.X11-unix
+METAMASK_DISPLAY=
+METAMASK_DISPLAY_VOLUME=
+METAMASK_SECRET_WORDS_OR_PRIVATEKEY=""
```

```suggestion
METAMASK_SECRET_WORDS_OR_PRIVATEKEY="test test test test test test test test test test test junk"
```
Can fill it in with test as placeholder